using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -58,17 +65,22 @@ ACPCampaign.extensionVersion().then(version => console.log("AdobeExperienceSDK:
 
 ## Set linkage fields
 
-Allows Campaign to connect fields from separate databases and create a more developed and personalized messaging experience. Profile template-based messages that contain PII-based personalization are downloaded. The linkage fields are stored as a base64-encoded JSON string in memory and sent in a custom HTTP header, `X-InApp-Auth`, in all future Campaign rules download requests until `ACPCampaign::resetLinkageFields` is invoked. The linkage fields are cleared when the app is gracefully closed, crashed, or the privacy status is changed to opt out. For more information, see [Preparing and sending an In-App message](https://helpx.adobe.com/campaign/standard/channels/using/preparing-and-sending-an-in-app-message.html).
+This API sets the Campaign linkage fields (CRM IDs) in the mobile SDK that are used to download personalized messages from Campaign. The set linkage fields are stored as a base64 encoded JSON string in memory, and they are sent in a custom HTTP header `X-InApp-Auth` in all future Campaign rules download requests until `resetLinkageFields` is invoked. These in-memory variables are also lost in the wake of an Application crash event or upon graceful Application termination or when the privacy status is updated to OPT_OUT. For more information, see [Preparing and sending an In-App message](https://helpx.adobe.com/campaign/standard/channels/using/preparing-and-sending-an-in-app-message.html).
```

Comment:
- Please capitalize "mobile" to "Mobile".

- Please reword the 3rd sentence as follows:
"These in-memory variables are lost when an application crash event occurs, after a graceful termination of the application, or when the privacy status is updated to `OPT_OUT`,